CHANGELOG-old.md

```diff
@@ -1 +1,40 @@
 # Historical Changelog
+
+
+## [rust-v0.6.0](https://github.com/delta-io/delta-rs/tree/rust-v0.6.0) (2022-12-16)
+
+[Full Changelog](https://github.com/delta-io/delta-rs/compare/rust-v0.5.0...rust-v0.6.0)
+
+**Implemented enhancements:**
+
+- Support Apache Arrow DataFusion 15 [\#1020](https://github.com/delta-io/delta-rs/issues/1020)
+- Python package: Loosen version requirements for maturin [\#1004](https://github.com/delta-io/delta-rs/issues/1004)
+- Remove `Cargo.lock` from library crates and add `Cargo.lock` to binary ones [\#1000](https://github.com/delta-io/delta-rs/issues/1000)
+- More frequent Rust releases [\#969](https://github.com/delta-io/delta-rs/issues/969)
+- Thoughts on adding read\_delta to pandas [\#869](https://github.com/delta-io/delta-rs/issues/869)
+- Add the support of the AWS\_PROFILE environment variable for S3 [\#986](https://github.com/delta-io/delta-rs/pull/986) ([fvaleye](https://github.com/fvaleye))
+
+**Fixed bugs:**
+
+- Azure SAS signatures ending in "=" don't work [\#1003](https://github.com/delta-io/delta-rs/issues/1003)
+- Fail to compile deltalake crate, need to update dynamodb\_lock in crates.io [\#1002](https://github.com/delta-io/delta-rs/issues/1002)
+- error reading delta table to pandas: runtime dropped the dispatch task [\#975](https://github.com/delta-io/delta-rs/issues/975)
+- MacOS arm64 wheels are generated incorrectly [\#972](https://github.com/delta-io/delta-rs/issues/972)
+- Overwrite creates new file [\#960](https://github.com/delta-io/delta-rs/issues/960)
+- The written delta file has corrupted structure [\#956](https://github.com/delta-io/delta-rs/issues/956)
```

Comment:
```suggestion
```

These two issues were just ones where users misunderstood how Delta tables works. They have no change set associated with them.

I wonder if the script that generates this could exclude issues that have no linked PRs?